.ci/integrationTestSelector.groovy

```diff
@@ -123,6 +125,8 @@ pipeline {
       }
       environment {
         TMPDIR = "${WORKSPACE}"
+        ENABLE_ES_DUMP = "true"
+        PATH = "${WORKSPACE}/${BASE_DIR}/.ci/scripts:${env.PATH}"
```

Comment:
Is it possible to move this section in the top-level environment?

Reply:
`WORKSPACE` and `PATH` are Agent dependent environment variables it could change, even do 99.99999% of times would be the same there is a possibility to not.

Reply:
Got it. Thanks